docs/source/examples/predict_next_purchase.ipynb

```diff
@@ -0,0 +1,400 @@
+    "* The `cutoff_time` as the label times that we generated previously."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "fm, fd = ft.dfs(\n",
+    "    entityset=es,\n",
+    "    target_entity='customers',\n",
+    "    cutoff_time=lt,\n",
+    "    cutoff_time_in_index=True,\n",
+    "    include_cutoff_time=False,\n",
+    "    verbose=False,\n",
+    ")\n",
+    "\n",
+    "fm.head()"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "\n",
+    "There are two outputs from DFS: a feature matrix and feature definitions. The feature matrix is a table that contains the feature values based on the cutoff times from our labels. Feature definitions are features in a list that can be stored and reused later to calculate the same set of features on future data.\n",
+    "\n",
+    "## Machine Learning\n",
+    "\n",
+    "In the previous steps, we generated the labels and features. The final step is to build the machine learning pipeline.\n",
+    "\n",
+    "### Splitting the Data\n",
+    "\n",
+    "We will start by extracting the labels from the feature matrix and splitting the data into a training set and holdout set."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "y = fm.pop('bought_product')\n",
+    "splits = evalml.preprocessing.split_data(fm, y, test_size=0.2, random_state=0)\n",
+    "X_train, X_holdout, y_train, y_holdout = splits"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### Finding the Best Model\n",
+    "\n",
+    "Then, we run a search on the training set for the best machine learning model."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "automl = evalml.AutoMLSearch(problem_type='binary', objective='f1', random_state=0)\n",
+    "automl.search(X_train, y_train, data_checks='disabled', show_iteration_plot=False)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Once the search is complete, we can print out information about the best pipeline found, such as the parameters in each component. "
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "automl.best_pipeline.describe()\n",
+    "automl.best_pipeline.graph()"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Now, let's score the model performance by evaluating predictions on the holdout set."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "best_pipeline = automl.best_pipeline.fit(X_train, y_train)\n",
+    "score = best_pipeline.score(X_holdout, y_holdout, objectives=['f1'])\n",
+    "dict(score)"
```

Comment:
maybe also show the top features using permutation performance or the prediction explanation functionality, just to show off evalml some more



this could be added as an add on later though

Reply:
The `graph_feature_importance` doesn't render for me in jupyter lab, but I can use the data frame for `feature_importance` to create a plot.

Reply:
it might render fine in the docs even if the jupyter doesnt work. this is also a known issue

Reply:
It doesn't render in the docs when I build locally.

Reply:
oh well. okay

Reply:
@jeff-hernandez @kmax12 you're probably running into this: https://github.com/FeatureLabs/evalml/issues/1040

@jeff-hernandez , do you have the `ipywidgets` pip package installed locally? I see you're listing `evalml>=0.11.2` in the requirements, which looks fine--that should include `ipywidgets`, and the graph should show up... perhaps you need to rerun pip install locally? Ping the team if you'd like some help with that.

Reply:
Oh also @jeff-hernandez : @freddyaboulton ran into this issue locally in jupyterlab, where he had to install [the jupyter lab extension](https://plotly.com/python/getting-started/) in order to get this working. If you use jupyterlab as opposed to jupyter, you'll probably also need to do this.